tools/pythonpkg/tests/fast/test_2747.py

```diff
@@ -0,0 +1,82 @@
+			, 'dec_18_3': [(Decimal('-999999999999.999999'),), (Decimal('999999999999.999999'),), (None,)], 'dec38_10':[(Decimal('-9999999999999999999999999999.9999999999'),), (Decimal('9999999999999999999999999999.9999999999'),), (None,)]
+			, 'uuid': [(UUID('00000000-0000-0000-0000-000000000001'),), (UUID('ffffffff-ffff-ffff-ffff-ffffffffffff'),), (None,)]
+			, 'varchar': [('🦆🦆🦆🦆🦆🦆',), ('goose',), (None,)], 'blob': [(b'thisisalongblob\x00withnullbytes',), (b'\\x00\\x00\\x00a',), (None,)]
+			, 'small_enum':[('DUCK_DUCK_ENUM',), ('GOOSE',), (None,)], 'medium_enum': [('enum_0',), ('enum_299',), (None,)], 'large_enum': [('enum_0',), ('enum_69999',), (None,)]
+			, 'int_array': [([],), ([42, 999, None, None, -42],), (None,)], 'varchar_array': [([],), (['🦆🦆🦆🦆🦆🦆', 'goose', None, ''],), (None,)]
+			, 'nested_int_array': [([],), ([[], [42, 999, None, None, -42], None, [], [42, 999, None, None, -42]],), (None,)], 'struct': [({'a': None, 'b': None},), ({'a': 42, 'b': '🦆🦆🦆🦆🦆🦆'},), (None,)]
+			, 'struct_of_arrays': [({'a': None, 'b': None},), ({'a': [42, 999, None, None, -42], 'b': ['🦆🦆🦆🦆🦆🦆', 'goose', None, '']},), (None,)]
+			, 'array_of_structs': [([],), ([{'a': None, 'b': None}, {'a': 42, 'b': '🦆🦆🦆🦆🦆🦆'}, None],), (None,)], 'map':[({'key': [], 'value': []},), ({'key': ['key1', 'key2'], 'value': ['🦆🦆🦆🦆🦆🦆', 'goose']},), (None,)] 
+			, 'time_tz':[(datetime.time(0, 0),), (datetime.time(23, 59, 59, 999999),), (None,)], 'interval': [(datetime.timedelta(0),), (datetime.timedelta(days=30969, seconds=999, microseconds=999999),), (None,)]}
+
+		for cur_type in all_types:
+			if cur_type not in skip_types:
+				result = conn.execute("select "+cur_type+" from test_all_types()").fetchall()
+				print (result)
+				correct_result = correct_answer_map[cur_type]
+				assert result == correct_result
+
+	def test_arrow(self, duckdb_cursor):
+		try:
+			import pyarrow as pa
+		except:
+			return
+		# We skip those since the extreme ranges are not supported in arrow.
+		skip_types = {'interval'}
+		# We do not round trip enum types
+		enum_types = {'small_enum', 'medium_enum', 'large_enum'}
+		conn = duckdb.connect()	
+		for cur_type in all_types:
+			if cur_type not in skip_types:
+				arrow_table = conn.execute("select "+cur_type+" from test_all_types()").arrow()
+				if cur_type in enum_types:
+					round_trip_arrow_table = conn.execute("select * from arrow_table").arrow()
+					result_arrow = conn.execute("select * from arrow_table").fetchall()
+					result_roundtrip = conn.execute("select * from round_trip_arrow_table").fetchall()
+					assert result_arrow == result_roundtrip
+				else:
+					round_trip_arrow_table = conn.execute("select * from arrow_table").arrow()
+					assert arrow_table.equals(round_trip_arrow_table, check_metadata=True)
+			
+	def test_pandas(self, duckdb_cursor):
+		# We skip those since the extreme ranges are not supported in python.
+		skip_types = {'date', 'timestamp', 'timestamp_s', 'timestamp_ns', 'timestamp_ms', 'date_tz', 'timestamp_tz'}
```

Comment:
Instead of skipping, can we do a replacement where we test the type but with a different value, e.g.:

```py
replacement_values = {
    'timestamp_s': "'1990-01-01 00:00:00'::TIMESTAMP_S"
    ...
}
```